tests/onnx/quantization/test_ptq_params.py

```diff
@@ -84,17 +94,20 @@ def test_params(self):
                 "nncf_graph": NNCFGraphToTest(
```

Comment:
This was done because most likely this test requires different NNCFGraphs for each backend BECAUSE backend specific NCNFGraphs have different functionality. If you check other backends you will see they are using different NNCFGraph.
NNCFGraphToTest was intorduced for `test_quantizer_config` and reused here.
Yes, maybe we can refactor it, but it won't bring any value at the end